ctools/defragment_sharded_collection.py

```diff
@@ -768,7 +774,7 @@ async def split_oversized_chunks(shard, progress):
                 continue
 
             local_c = chunks_id_index[c['_id']]
-            if local_c['defrag_collection_est_size'] > target_chunk_size_kb * 1.6:
+            if local_c['defrag_collection_est_size'] > target_chunk_size_kb * 2.4:
                 await coll.split_chunk(local_c, target_chunk_size_kb)
```

Comment:
Taking into account that `splitVector` command returns splits points at half `target_chunks_size`, wouldn't be better to pass `target_chunk_size_kb * 2` here ?

Reply:
Totally right, fixed!